/jds/bin/diff -uprN libmikmod-3.2.0-beta2.old/docs/libmikmod-config.1.in libmikmod-3.2.0-beta2/docs/libmikmod-config.1.in
--- libmikmod-3.2.0-beta2.old/docs/libmikmod-config.1.in	2009-02-20 19:03:16.651385000 +0000
+++ libmikmod-3.2.0-beta2/docs/libmikmod-config.1.in	2009-02-20 19:06:30.905634000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .TH LIBMIKMOD 1 "Version @VERSION@, 3rd Januark 2000"
 
 .SH NAME
@@ -46,6 +47,21 @@ If specified, use PREFIX instead of the 
 \-\-libs options.  This option must be specified before any \-\-libs or
 \-\-cflags options.
 
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWlibmikmod
+Interface stability^Uncommitted
+.TE
+.sp
+
 .SH AUTHORS
 \fIlibmikmod\fP is the result of the work of many people, including:
 Jean-Paul Mikkers, Jake Stine, Miodrag Vallat, Steve McIntyre, Peter Amstutz,
@@ -57,3 +73,4 @@ source package.
 This man page was inspired by the gtk-config man page written by Owen Taylor.
 
 The k in ``Januark'' is not a typo. It's a genuine Y2K bug.
+...\" LSARC 2008/422 libmikmod
